modules/integration/tests-integration/tests-backend/src/test/resources/ws-trust-templates/request_security_token_RST.xml

```diff
@@ -0,0 +1,56 @@
+<soapenv:Envelope xmlns:soapenv="http://www.w3.org/2003/05/soap-envelope">
```

Comment:
can we add license headers to these resource files?

Reply:
These files are used for the test if we add the license header then the tests would fail.